packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -106,6 +110,23 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
         this.unitController.onViewRangeChanged(viewRangeParam => { this.handleViewRangeChange(viewRangeParam); });
         this.traceContextContainer = React.createRef();
         this.initialize();
+        signalManager().on(Signals.THEME_CHANGED, (theme: string) => this.updateBackgroundTheme(theme));
+    }
+
+    public updateBackgroundTheme(theme: string) {
+        this.setState({
+            style: {
+                width: this.DEFAULT_COMPONENT_WIDTH,
+                chartWidth: this.DEFAULT_CHART_WIDTH,
+                height: this.DEFAULT_COMPONENT_HEIGHT,
+                rowHeight: this.DEFAULT_COMPONENT_ROWHEIGHT,
+                naviBackgroundColor: theme === 'light' ? 0xf5f7f7 : 0x3f3f3f,
+                chartBackgroundColor: theme === 'light' ? 0xf5f7f7 : 0x3f3f3f,
+                cursorColor: 0x259fd8,
+                lineColor: theme === 'light' ? 0x8b8c8c : 0xbbbbbb
```

Comment:
It could be nice to extract these style methods to somewhere central, so there would be a single goto place to implement new themes or change values for an existing theme.

Reply:
Agree. I would like to fix it as part of another PR. Thank you for the suggestion :)